package.json

```diff
@@ -28,7 +32,9 @@
     "@types/gunzip-maybe": "^1.4.0",
     "@types/rimraf": "^3.0.2",
     "@types/tar-stream": "^2.2.2",
+    "eslint": "^8.0.0",
```

Comment:
Please remove `eslint` and the lint commands -  between TypeScript's compiler and prettier's formatter, we should be well covered. Thanks!